### ISSUE TITLE
Launchbuddy Multiclient

### ISSUE THREAD

**Issue by Zaidya** [megai2/d912pxy#346]
**Describe the bug**
I started using Launchbuddy to multiclient 2 accounts. There I had the bug, that the first account started without problems, but the 2nd one didn't start and I got an gw2 error after a fiew minutes for it. I uninstalled everything, addons, launchbuddy, gw2 and installed it clean and new. Then I added one addon after another to figure, wich could cause the problem (because friends haven't had those kind of crashes). I ended up finding the error again after I added d912pxy with the addon manager. Second Client didn't start again and a few minutes later I got the gw2 error "something went wrong, please contact support". No clear error number or description. I also can just close the launchbuddy boxes as if gw2 wouldn't run with them. 

**To Reproduce**
Have 2 gw2 accounts
setup Launchbuddy for both accounts
have addons installed on normal gw2 folder with addon manager 
start both accounts on launchbuddy together
Friends said, sometimes it works for them, for me it doesn't.

**Expected behavior**
Supporting multiclienting with 2 or more clients, using the directX12 Tool from you. 

**Screenshots**

**Standard information**

1. Windows 10 20H2
2. Hardware info:  Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz   3.70 GHz, 16GB 
3. GPU Driver version - AMD Radeon R9 200 Series - Driver Version 27.20.1034.6
4. Version of d912pxy used - downloaded current one with addonmanager
5. Log file from <game root>/P7logs folder or d912pxy/log.txt and d912pxy/crash.txt if exist 
[log.txt](https://github.com/megai2/d912pxy/files/6097851/log.txt)
![gw2 error](https://user-images.githubusercontent.com/69413243/110253023-6a433000-7f88-11eb-94d4-cf6de7f0d10b.png)

**Additional context**
gw2 error file translation: "Installation could not be completed. Please contact support."

**Comment by beng1302** [comment]
This also happens using gw2launcher after updating to v2.4.3.1. A current workaround is to copy the entire gw2 folder for each account under gw2launcher->settings->gw2->management->desired level=full. This will copy the full 50gb gw2.dat and all addon folders too which is hardly ideal.

**Comment by Caelus4058** [comment]
Got the same issue sadly. Is there a workaround for the launchbuddy ppl available atm?

**Comment by SaiyanOfDarkness** [comment]
Does it work if just launching the game using the original client?

**Comment by Zaidya** [comment]
> Does it work if just launching the game using the original client?

Yeah, but I want to multi client, that's why I was reporting this bug *lol* ofc the game works with the normal client.

**Comment by Caelus4058** [comment]
> Does it work if just launching the game using the original client?

The initial / first client works. With the second Client starting, you won't even see a loading screen. Basically log in works, you press play and the next window is the error code...

**Comment by seanbarr1988** [comment]
I get nothing working not even the first client. i dont see anything until a long wait then the window appears

**Comment by AEsylum** [comment]
Issue not resolved with 2.4.4

**Comment by unrelentingsarcasm** [comment]
I get this issue only when trying to also run GW2 Radial.  My set up with LaunchBuddy is to use ArcDPS as d3d9.dll in the bin64 folder, d912pxy.dll in the bin64 folder, GW2Hook as ReShade64.dll in the bin64 folder.  For each of these I load them in order in LB's injected software account setting and I have no issues or crashes.  However, any attempt to add in GW2Radial and it will not load d912pxy and the game won't load for even 1x account.  I am currently using d912pxy v2.4.4_r696 and the new GW2Radial v2.1.0-pre1 that is supposed to have better d912pxy integration.

My goal is to multibox with LB while retaining Arc, d912pxy and the look of GW2Hook, and ideally run Radial for the ease of selecting mounts, commander markers, and novelties.

The error above appears to occur when I force GW2Radial to load before d912pxy through changing the name to d3d9_chainload or prioritizing it ahead of d912pxy in the injected software queue.  But I get the issue displayed in the following link when I attempt to inject the Radial .dll after the d912pxy.dll.

https://github.com/Friendly0Fire/GW2Radial/issues/210

Is this issue resolvable?
